docs/compendium/substitute.md

```diff
@@ -1,31 +1,107 @@
-# Substitute compendia
+# Substitute two compendia
 
-introduction...
+Substitution is the combination of an base erc and an overlay erc.
+A user can choose files from the overlay erc that will be replaced with files of the base erc or uniquely added.
 
 ## Create substitution
 
 ### Request
 
-`POST /api/v1/..`
+`POST /api/v1/substitution`
+
+input of request-body for substitution
+```json
+{
+  "base": "base_erc_id",
+  "substFiles": [
+    {
+      "original": "base_datei",
+      "xchange": "overlay_datei"
+    }
+  ],
+  "overlay": "overlay_erc_id",
+}
+```
+
+### body parameters for substititions
+
+- `base` - id of the base erc
+- `substfiles` - array objects specified by `original` and `xchange`
+  - `original` - filename of the file from the base erc
+  - `xchange` - filename of the overlay erc that will be exchanged for the original file
+- `overlay` - id of the overlay erc
 
 ### Response
 
 ```json
-200 OK
+201 CREATED
 
 {
+  "id": "new_erc_id"
 }
 ```
 
-### URL parameters for substititions
+### Error responses
 
-- `base` ?
-- `overlay` ?
+```json
+401 Unauthorized
 
-### Error responses
+{"error":"not authenticated"}
+```
+
+```json
+401 Unauthorized
+
+{"error":"not allowed"}
+```
 
 ```json
 404 Not Found
 
 {"error":"base erc not found"}
 ```
+
+```json
+404 Not Found
+
+{"error":"overlay erc not found"}
+```
+
+```json
+500 Internal Server Error
+
+{"error":"Error during substitution"}
+```
+
+## View single Compendium
```

Comment:
motivation: a substituted ERC will be handled as a normal (uploaded) ERC and there is also an explanation of what will be added to the metadata of a substituted ERC.